src/server/game/Globals/ObjectMgr.h

```diff
@@ -1244,6 +1244,12 @@ class TC_GAME_API ObjectMgr
         void LoadSpawnGroups();
         void LoadInstanceSpawnGroups();
         void LoadItemTemplates();
+        // @tswow-begin
+        void LoadSingleItemTemplate(std::string entry);
```

Comment:
It looks like this function isn't being used by anything, maybe that's why it was left with an incorrect loop while the other one was correct.

It's also weird to use a string as an argument, if you want to keep it that should be a uint32.